tck/src/main/java/org/eclipse/configjsr/ConverterTest.java

```diff
@@ -85,6 +87,19 @@ public static WebArchive deploy() {
     @ConfigProperty(name = "tck.config.test.javaconfig.converter.duckname")
     private Duck namedDuck;
 
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testDonaldNotConvertedByDefault() {
+        config.getValue("tck.config.test.javaconfig.converter.donaldname", Donald.class);
```

Comment:
This test will no longer fail when we support implicit converters as the Donald class as a `Donald(String)` constructor.
I'll fix it in the PR to ensure that this class will not match such an implicit converter